controllers/clusters/kafka_controller.go

```diff
@@ -234,9 +236,180 @@ func (r *KafkaReconciler) handleUpdateCluster(
 		return models.ReconcileRequeue
 	}
 
+	for _, ref := range k.Spec.UserRefs {
+		err = r.handleCreateUser(ctx, k, l, ref)
+		if err != nil {
+			l.Error(err, "Cannot create Kafka user", "user", ref)
+			r.EventRecorder.Eventf(k, models.Warning, models.CreatingEvent,
+				"Cannot create user. Reason: %v", err)
+		}
+	}
+
 	return models.ExitReconcile
 }
 
+func (r *KafkaReconciler) handleCreateUser(
+	ctx context.Context,
+	kafka *v1beta1.Kafka,
+	l logr.Logger,
+	userRef *v1beta1.UserReference,
+) error {
+	req := types.NamespacedName{
+		Namespace: userRef.Namespace,
+		Name:      userRef.Name,
+	}
+
+	u := &kafkamanagementv1beta1.KafkaUser{}
+	err := r.Get(ctx, req, u)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			l.Info("kafka user is not found", "request", req)
+			r.EventRecorder.Event(kafka, models.Warning, "Not Found",
+				"User is not found, create a new one Kafka User or provide correct userRef.")
```

Comment:
`User is not found, create a new one Kafka User or provide correct userRef.` -> `Kafka user is not found, create a new one or provide correct userRef.`

Reply:
done